src/dali/plugin/input/rest/bitbank.py

```diff
@@ -103,3 +117,34 @@ def _process_implicit_api(
         intra_transactions: List[IntraTransaction],
     ) -> None:
         pass
+
+    # Override the handling of trades since Bitbank has negative maker fees
+    def _process_buy_and_sell(self, transaction: Any, notes: Optional[str] = None) -> ProcessOperationResult:
+        fee_income: Optional[InTransaction] = None
+        if RP2Decimal(str(transaction[_FEE][_COST])) < ZERO:
+            fee_income = InTransaction(
+                    plugin=self.plugin_name(),
+                    unique_id=f"FI{transaction[_ID]}",
```

Comment:
Why the prepended "FI": wouldn't it throw off the transaction resolver if we mangle the unique id?

Reply:
I guess for some reason I thought it would confuse the transaction resolver. I guess 'fees' like this won't bother it.